canonical-websites/www.canonical.com
# Partners: css is linted and remove legacy css

Done:
Linted, cleaned up, and stripped unnecessary styling.

QA:
Ensure that the /partners page renders correctly

Card:
https://canonical.leankit.com/Boards/View/111185042/119259236
